src/resize-sensor/ResizeSensor.d.ts

```diff
@@ -0,0 +1,14 @@
+import { PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export type ResizeSensorProps = {
+  children: any,
+  onResize: (elements: HTMLElement) => void,
```

Comment:
yeah, but your should look at `this.props.onResize` inside the component and right now it's called with no arg